trello_app.py

```diff
@@ -0,0 +1,121 @@
+from flask import Blueprint
+from flask import current_app as app
+from flask import Flask, render_template, request, redirect, url_for
+import config as cf
+from model import Item, ViewModel
+
+import session_items as session
+import requests
+import json
+
+trello_bp = Blueprint( 'trello_bp', __name__)
+
+@trello_bp.route('/', methods=['GET'])
+#@app.route('/')
+def index():
+    board_id = cf.get_trello_board_id()
+    url = f"https://api.trello.com/1/boards/{board_id}/cards"
+    query = cf.get_trello_query()
+    
+    response = requests.request(
+        "GET",
+        url,
+        params=query
+        )
+    
+    cards = json.loads(response.text)
+    items_list = list()
+    for card in cards:
+        status = 'To Do' 
+        if card['idList'] == cf.get_trello_list_id():
+            status = 'To Do'
+        elif card['idList'] == cf.get_trello_list_id_doing():
+            status = 'Doing'
+        else:
+            status = 'Done'
+
+        item = Item(card['id'], status, card['name'], card['dateLastActivity'])
+        items_list.append(item)
+
+    item_view_model = ViewModel(items_list)
+    #render_template('index.html', view_model=item_view_model)
+
+    #return render_template('index.html', items=items_list)
+    return render_template('index.html', view_model=item_view_model)
+
+
+@trello_bp.route('/complete_item/<idCard>', methods=['GET', 'PUT'])
+#@app.route('/complete_item/<idCard>', methods=['GET', 'PUT'])
+def update_card(idCard):
+
+    url = "https://api.trello.com/1/cards/{idCard}"
```

Comment:
This line can never work. You need to make it a format string:
```suggestion
    url = f"https://api.trello.com/1/cards/{idCard}"
```